Tests/TestUtilities/DummyExeHelper.cs

```diff
@@ -35,18 +35,33 @@ public static string CreateDummyPostProcessor(string dummyBinDir, int exitCode)
             return CreateDummyExe(dummyBinDir, PostProcessorExeName, exitCode);
         }
 
+        public static string CreateDummyExe(string outputDir, string exeName, int exitCode)
```

Comment:
The _DummyExeHelper_ helper class creates a simple command line exe on the fly that returns a specified error code and logs the command line arguments that were passed to it. It was used by bootstrapper tests that launched the pre- and post- processor. I've made it slightly more general and added the the ability to specify additional code to be added to the exe to enable testing other scenarios (e.g. time outs and errors).